src/components/popups/popupStructure/popupStructure.css

```
@@ -0,0 +1,357 @@
+.structure-morpheme {
+  position: relative;
+  padding-top: 12px;
+  white-space: pre;
+}
+
+/* основа: everything except the ending is underlined */
+.structure-morpheme-prefix,
+.structure-morpheme-root,
+.structure-morpheme-suffix,
+.structure-morpheme-postfix {
+  border-bottom: 2px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)